HW10/read.txt

```diff
@@ -0,0 +1,101 @@
+var b = [5, 10, 23, 10, 7, 9, 39, 16, 1, 12, 36, 20, 50];
```

Comment:
so its right , do you want me to correct ruby also and push it again through github